### PR TITLE
Switch to git-sig for signing/verification

### PR DESCRIPTION
**Pull request by lrvick**
- [x] cache all pgp keys used for verification in repo
- [x] drop manifest.txt as we rely on git tree hashes now
- [x] update readme to reflect new workflow

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -66,6 +48,9 @@ linux-config:
 	rm $(CONFIG_DIR)/$(TARGET)/linux.config
 	make TARGET=$(TARGET) $(CONFIG_DIR)/$(TARGET)/linux.config
 
+$(KEY_DIR)/%.asc:
```
Comment (emostov) [inline Makefile:51]: what does this do?

File: README.md [hunk 2]
```diff
@@ -41,45 +41,73 @@ security applications.
...
+
+#### Signatures
+
+Once you have public keys you trust locally pinned, you are able verify that
+the artifacts and code we publish are validly signed.
+
+We require a minimum of 2 signatures so you can use [git-sig][gs] as follows:
+
+```
+git sig verify --threshold 2
```
Comment (emostov) [inline README.md:101]: can we add a section on how to install git sig

File: Makefile (hunk 1 above)
Comment (lrvick) [reply Makefile:51]: This pulls down copies of public keys for the local cache so people have a super fast path to verify (as you can't do that without public keys).

File: README.md (hunk 2 above)
Comment (lrvick) [reply README.md:101]: All the steps on installing git-sig are in the git-sig readme, as with git-lfs.

Copy-pasting all of that into every project felt like a lot but open to ideas.

**Pull request merged**